src/styles/pages/checkout/checkout.scss

```diff
@@ -94,6 +94,42 @@
         padding-left: $space-default * 2;
       }
     }
+
+    &.infobox-color-widget {
+      padding-bottom: 15px;
+      font-size: 14px;
+      color: $text-color-inverse;
+      text-transform: uppercase;
+      background: $CORPORATE-PRIMARY;
+      h3 {
+        font-size: 1.09375rem;
+        color: $text-color-inverse;
+      }
+      a {
+        color: $text-color-inverse;
+        text-decoration: underline;
+        text-transform: none;
+      }
+      .infobox-count {
+        font-size: 36px;
+        line-height: 1;
+      }
+      .infobox-count-large {
+        font-size: 60px;
+        line-height: 1;
+      }
+      .row:last-of-type {
+        margin-top: 0.5rem;
+      }
+    }
+
+    &.infobox-widget {
+      padding-bottom: 15px;
+
+      .row:last-of-type {
```

Comment:
why is this necessary - i couldn`t recognize a difference

Reply:
This is necessary for more space between the budget widget content and link at the bottom